src/system_info/system_info_network_ivi.cc

```diff
@@ -18,6 +18,8 @@ SysInfoNetworkTizen::~SysInfoNetworkTizen() {
       cellular_network_type_changed_watch_);
   g_dbus_connection_close_sync(conn_, NULL, NULL);
   conn_ = NULL;
+  free(modem_path_);
+  modem_path_ = NULL;
```

Comment:
First of all, thanks for catching up the memory leak bug.

Read through this PR, I'd suggest:
1. Change `SysInfoNetworkTizen::modem_path_` to `std::string`? 
2. Change `OfonoGetModemPath()` return `std::string` instead of `char*`

Thus we can avoid allocated memory in `OfonoGetModemPath()` and free somewhere else (`SysInfoNetworkTizen::~SysInfoNetworkTizen()`).